lib/cisco_node_utils/cmd_ref/tacacs_global.yaml

```diff
@@ -3,6 +3,13 @@
 encryption:
   set_value: "<state> tacacs-server key <option> <key>"
 
+feature:
```

Comment:
We already have an entry for this in `feature.rb` and `feature.yaml`.  Please use these instead of adding another entyr in `tacacs_global.yaml` and `tacacs_global.rb`

- https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/feature.yaml#L71
- https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/feature.rb#L195